payment-settler/wrangler.toml

```diff
@@ -0,0 +1,112 @@
+name = "filcdn-payment-settler"
+main = "bin/payment-settler.js"
+compatibility_date = "2024-12-05"
+compatibility_flags = ["nodejs_compat"]
+logpush = true
+
+[observability]
+enabled = true
+head_sampling_rate = 1
+
+[[d1_databases]]
+binding = "DB"
+database_name = "test-db"
+database_id = "8cc92155-16f6-426a-b782-2965e0daf100"
+
+[[queues.producers]]
+queue = "transaction-queue"
+binding = "TRANSACTION_QUEUE"
+
+[[queues.consumers]]
+queue = "transaction-queue"
+max_batch_size = 10
+max_batch_timeout = 30
+
+[[workflows]]
+binding = "TRANSACTION_MONITOR_WORKFLOW"
+name = "transaction-monitor-workflow"
+class_name = "TransactionMonitorWorkflow"
+
+[env.dev.vars]
+ENVIRONMENT = "dev"
+RPC_URL = "https://api.calibration.node.glif.io/"
+FILBEAM_CONTRACT_ADDRESS = "0x96e13bC98fA0cdf01b07985c804960117EAB76E8"
+DEFAULT_LOCKUP_PERIOD_DAYS = 30 # 30 days lockup period
+FILECOIN_GENESIS_BLOCK_TIMESTAMP_MS = 1667326380000 # Calibration network genesis timestamp in milliseconds
+
+[[env.dev.d1_databases]]
+binding = "DB"
+database_name = "dev-db"
+database_id = "8cc92155-16f6-426a-b782-2965e0daf101"
+
+[[env.dev.queues.producers]]
+queue = "dev-transaction-queue"
+binding = "TRANSACTION_QUEUE"
+
+[[env.dev.queues.consumers]]
+queue = "dev-transaction-queue"
+max_batch_size = 10
+max_batch_timeout = 30
+
+[[env.dev.workflows]]
+binding = "TRANSACTION_MONITOR_WORKFLOW"
+name = "dev-transaction-monitor-workflow"
+class_name = "TransactionMonitorWorkflow"
+
+[env.calibration.vars]
+ENVIRONMENT = "calibration"
+RPC_URL = "https://api.calibration.node.glif.io/"
+FILBEAM_CONTRACT_ADDRESS = "0x"
```

Comment:
Missing contract address.

Should we wait with landing this PR until there is a FilBeamOperator version deployed to calibration?

Alternatively, how about changing the cron schedule so that this worker is never executed?